zwave_js/rootfs/etc/services.d/zwave_js/run

```diff
@@ -12,5 +12,5 @@ fi
 # Send out discovery information to Home Assistant
 ./discovery &
 
-# Run daemon
-exec zwave-server ${SERIAL_DEVICE} --config /etc/zwave_config.json
+# Run daemon, passing external config directory in as environment variable
+exec env ZWAVEJS_EXTERNAL_CONFIG=/data/db zwave-server ${SERIAL_DEVICE} --config /etc/zwave_config.json
```

Comment:
A little more readable suggestion

```suggestion
# Set external config directory
export ZWAVEJS_EXTERNAL_CONFIG=/data/db

# Run daemon, passing external config directory in as environment variable
exec zwave-server ${SERIAL_DEVICE} --config /etc/zwave_config.json
```

Reply:
I swear I tried this and it didn't work, but of course now it does 😅